kuma/users/tests/test_templates.py

```diff
@@ -409,3 +411,263 @@ def test_common_reasons_empty(self):
         eq_(len(reasons_to_ban_found), len(reasons_to_ban_expected))
         for reason in reasons_to_ban_found:
             ok_(reason.text in reasons_to_ban_expected)
+
+
+@pytest.mark.bans
+class BanAndCleanupTestCase(UserTestCase):
+    def test_user_revisions_in_one_click_page_template(self):
+        """The user's revisions show up in the ban and cleanup template."""
+        testuser = self.user_model.objects.get(username='testuser')
+        admin = self.user_model.objects.get(username='admin')
+
+        # Create an original revision on a document by the admin user
+        document = create_document(save=True)
+        original_revision = create_revision(
+            title='Revision 0',
+            document=document,
+            creator=admin,
+            save=True)
+
+        # Create 3 revisions for testuser, titled 'Revision 1', 'Revision 2'...
+        revisions_expected = []
+        num_revisions = 3
+        for i in range(1, 1 + num_revisions):
+            new_revision = create_revision(
+                title='Revision {}'.format(i),
+                document=document,
+                creator=testuser,
+                save=True)
+            revisions_expected.append(new_revision)
+
+        self.client.login(username='admin', password='testpass')
+        ban_url = reverse('users.ban_user_and_cleanup',
+                          kwargs={'user_id': testuser.id})
+
+        resp = self.client.get(ban_url, follow=True)
+        eq_(200, resp.status_code)
```

Comment:
Someday (2017?) I'll write that CONTRIBUTING doc about "stop using ``eq_`` and ``ok_``, just use ``assert``. For now, use whatever style is most common in the existing tests in the file.